dwcj-engine/src/main/java/org/dwcj/component/choicebox/ChoiceBox.java

```diff
@@ -681,5 +748,12 @@ protected void catchUp() throws IllegalAccessException {
             this.selectIndex(this.selected);
         }
 
+        if(this.placeholder.equals("")){
```

Comment:
Please change these to check for inequality:

if(!this.placeholder.equals("")){ 
...